src/content/install/troubleshoot.md

```diff
@@ -170,6 +177,47 @@ To fix this:
    launch Flutter apps.
    You might need to restart your code editor, terminal, and physical device.
 
+### Problem running the iOS Simulator
+
+Set your Simulator to use a 64-bit device.
+
+   * From **Xcode**, choose a simulator device type.
+
+     1. Go to **Window** <span aria-label="and then">></span>
+        **Devices and Simulators** or press
+        <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>2</kbd>.
+
+     2. Once the **Devices and Simulators** dialog opens,
+        click **Simulators**.
+
+     3. Choose a **Simulator** from the left-hand list or
+        press **+** to create a new simulator.
+
+   * From the **Simulator** app, go to
+     **File** <span aria-label="and then">></span>
+     **Open Simulator** <span aria-label="and then">></span>
+     Choose your target iOS device.
+
+   * To check the device version in the Simulator,
+     open the **Settings** app <span aria-label="and then">></span>
+     **General** <span aria-label="and then">></span>
+     **About**.
+
+### Window overflow using Simulator
+
+If you selected a simulated high-screen density iOS device,
+it might overflow your screen. If that occurs,
+change the presented size in the **Simulator** app.
```

Comment:
I also think this should be removed, it's a very niche situation to be in, I've never seen a customer reported issue of this (which doesn't mean there isn't one, but I think it's so uncommon as to be confusing to even mention).

Reply:
I think that makes sense. I'd rather avoid any extra confusion. I've removed this and the 64-bit troubleshooting sections. If we get reports of these issues, we can reconsider adding a new entry here.